src/WordSearch.java

```diff
@@ -0,0 +1,33 @@
+import java.io.File;
+
+/**
+ * Getting the file path from the command prompt
+ * calling the filePath method for searching the file in system
+ */
+public class WordSearch {
+    public static void main(String[] args) {
+        if (args.length == 2) {
```

Comment:
As per requirement, only txt and json files should be supported. Please add a check for that as well.

Reply:
Updated to check .txt and .json file format.